views/index.pug

```diff
@@ -84,5 +85,30 @@ block content
                       label Image Url
                       input.form-control(type='text', value=book.image_url, name='image_url', id="edit-form-image")
                   .modal-footer
-                    button.btn.btn-default(type='button', data-dismiss='modal') Close
-                    button.btn.btn-primary(type='submit', value='save' ) Save
+                          button.btn.btn-default(type='button', data-dismiss='modal') Close
+                          button.btn.btn-primary(type='submit', value='save' ) Save
+
+          .modal.fadeç(id=`description${book.id}Modal`, tabindex='-1', role='dialog', aria-labelledby='descriptionModalLabel', aria-hidden="true")
+            .modal-dialog(role='document')
+              .modal-content.closeDescription
+                    .modal-header
+                      button.close(type='button', data-dismiss='modal', aria-label='Close')
+                        span(aria-hidden='true') ×
+                      h4#descriptionModalLabel.modal-title=book.title
+                    .modal-body.description
+                      .form-group.sectionOne
+                        .leftContainer
+                            img.detailsBookCover(src=book.image_url, alt='book cover')
+                        .rightContainer
+                            p=`Author: ${book.author}`
+                            p=`Genre: ${book.genre}`
+                            br
```

Comment:
Consider using CSS to manage spacing - `br` can be a little difficult to manage since it's line height is related to the font size, meaning that you can only position elements in increments of the font-size in use in the current block.  Applying margin or padding to an element (in this case the `button`) allows you to explicitly control spacing on the page.